node-repository/src/main/java/com/yahoo/vespa/hosted/provision/maintenance/AutoscalingMaintainer.java

```diff
@@ -55,23 +55,32 @@ protected double maintain() {
         for (var applicationNodes : activeNodesByApplication().entrySet()) {
             for (var clusterNodes : nodesByCluster(applicationNodes.getValue()).entrySet()) {
                 attempts++;
-                if ( ! autoscale(applicationNodes.getKey(), clusterNodes.getKey()))
-                    failures++;
+                Result result = autoscale(applicationNodes.getKey(), clusterNodes.getKey());
```

Comment:
I think this is fine, but why not do it simply by doing the deploy attempt outside the try-with block in autoscale?